client/web/src/components/externalServices/ExternalServicePage.tsx

```diff
@@ -262,17 +262,17 @@ export const ExternalServicePage: FC<React.PropsWithChildren<Props>> = props =>
                             telemetryService={telemetryService}
                         />
                     )}
+                    <ExternalServiceWebhook externalService={externalService} className="mt-3" />
                     <LoaderButton
                         label="Trigger manual sync"
-                        className="mt-3"
+                        className="mt-2 mb-2 float-right"
```

Comment:
I'd leave mt-3, looks a bit better to my taste, but it's my taste...

```suggestion
                        className="mt-3 mb-2 float-right"
```